modules/replicator/src/main/java/org/apache/ignite/internal/replicator/ReplicaManager.java

```diff
@@ -292,14 +292,17 @@ public ReplicaManager(
                 NamedThreadFactory.create(nodeName, "scheduled-idle-safe-time-sync-thread", LOG)
         );
 
-        throttledLogExecutor = new ThreadPoolExecutor(
+        var executor = new ThreadPoolExecutor(
```

Comment:
Maybe we can do without the extra variable with a generalized name? Or rename it to `throttledLogExecutorInternal`.